docs/py_docs.rst

```diff
@@ -838,35 +1042,73 @@ At minimum, constants/properties/attributes should have a summary line, but can
 5. :ref:`References <py-docstring-references>` (optional)
 6. :ref:`Examples <py-docstring-examples>` (optional)
 
-In the short summary, a description of the type should be included:
+For example:
 
 .. code-block:: py
 
    NAME = 'LSST'
-   """Name of the project (str)"""
+   """Name of the project (`str`)."""
 
-Note that class attributes can alternatively be documented in an :ref:`Attributes <py-docstring-attributes>` section of the class's docstring.
-This is particularly useful when the attribute is not set in the class scope, but rather in a method such as ``__init__``.
+Note:
 
-.. code-block:: py
+- The docstring appears directly below the constant or class attribute.
+- The type is included in parentheses at the end of the summary line.
```

Comment:
Consider an example with more than just a summary line (I assume the type would still be at the end of the first line, not the end of the docstring).